wikipedia-dark.user.css

```diff
@@ -471,10 +471,10 @@
   .mw-body-content .portal ul,
   .wikitable > tr,
   .wikitable > tr > th,
-  .wikitable > tr > td,
-  .wikitable > * > tr,
+  .wikitable > tr > td:not([style*="background-color"]),
+  .wikitable > * > tr:not([bgcolor]),
   .wikitable > * > tr > th,
-  .wikitable > * > tr > td:not(.break),
+  .wikitable > * > tr > td:not(.break):not([style*="color"]):not([style*="background"]):not([bgcolor]),
   .wikitable th span[style*="background"],
```

Comment:
This has a high potential for collateral damage, especially given the amount of custom wikitable colors and styles employed on Wikipedia. Until they have more specific fixes, this rule remains necessary.

I would be hesitant to accept this change without reviewing quite a few wikitables with color scales to ensure it has the desired effect. We man find this starts an endless game of whack-a-mole.